runtime/src/main/java/com/tietoevry/quarkus/resteasy/problem/jaxrs/WebApplicationExceptionMapper.java

```diff
@@ -22,10 +23,10 @@ protected HttpProblem toProblem(WebApplicationException exception) {
                 .withStatus(status)
                 .withDetail(exception.getMessage());
 
-        exception
-                .getResponse()
-                .getHeaders()
-                .forEach((header, values) -> values.forEach(value -> problem.withHeader(header, value)));
+        Optional.ofNullable(exception.getResponse().getHeaders())
+                .ifPresent(headers -> {
```

Comment:
`ifPresent` is [passe](https://dzone.com/articles/optional-ispresent-is-bad-for-you) (right from the design) 😉 

Prefer
```java
Optional.ofNullable(...).orElse(<empty>).forEach(h -> ...)
```

Reply:
`ifPresent` != `isPresent` :P I see nothing wrong with `ifPresent`, it's much more readable (and obvious) than e.g `flatMap` for most developers, including me...